inspirehep/modules/forms/static/js/forms/experiments_typeahead.js

```diff
@@ -32,12 +32,14 @@ define([
       remote: {
         url: '/api/experiments?q=experimentautocomplete:%QUERY*',
         filter: function(response) {
-          return $.map(response.hits.hits, function(el) { return el });
-        }
-      },
+          return $.map(response.hits.hits, function(el) { return el }).sort(function(x, y) {
```

Comment:
Ah, this was introduced in https://github.com/inspirehep/inspire-next/pull/1602, but there's absolutely no point in doing it...